doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

```diff
@@ -159,3 +200,173 @@ Similarly, MCUboot uses the key file designated by the :Kconfig:option:`SB_CONFI
 At the end of the described process, the :file:`keyfile.json` file is generated in the build directory.
 This file allows key provisioning to occur simultaneously with the flashing process.
 Alternatively, you can bypass the mentioned Kconfig options and manually place a custom :file:`keyfile.json` in the build directory.
+
+.. _ug_nrf54l_developing_provision_kmu_psa_crypto_api:
+
+Provisioning KMU with PSA Crypto API
+************************************
+
+You can use the PSA Crypto API to create a provisioning image to be used to provision keys to the device during development or production.
+
+Detailed steps for creating such a provisioning image for your application are beyond the scope of this guide.
+Use the guidelines in :ref:`ug_nrf54l_crypto_kmu_key_programming_model` and in the `nRF54L Series Production Programming`_ guide.
+
+.. _ug_nrf54l_developing_provision_kmu_production:
+
+Provisioning KMU with production tools
+**************************************
+
+For production, `provision the KMU data <Provisioning KMU data_>`_ as described in the `nRF54L Series Production Programming`_ guide.
+
+Nordic Semiconductor recommends using other tools than the ones used for :ref:`provisioning for development <ug_nrf54l_developing_provision_kmu_development>`.
+For example, you could use SEGGER J-Link or other debugging tools.
+
+Using other tools means that you need to manually make sure that the values set in the :c:struct:`kmu_metadata` data structure's bitfields match the required values for different key types and the CRACEN driver's own requirements.
```

Comment:
How about:
Using other tools means that you need to manually make sure that the values stored in the KMU slots metadata field aligns with the CRACEN driver's :c:struct:`kmu_metadata` data structure's expectations for different key types.